src/node/internal/module.d.ts

```diff
@@ -2,5 +2,6 @@
 // Licensed under the Apache 2.0 license found in the LICENSE file or at:
 //     https://opensource.org/licenses/Apache-2.0
 
-export function createRequire(path: string): (specifier: string) => unknown;
-export function isBuiltin(specifier: string): boolean;
+export function createRequire(
+  path: string | URL
```

Comment:
This should be path: string. The previous code was correct.

Reply:
oh right that's the internal module, I'll revert, thanks for catching!